Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -289,6 +289,41 @@
       zeroVisible: true
     - type: Appearance
 
+- type: entity
+  name: practice disabler
+  parent: BaseWeaponBatterySmall
+  id: WeaponDisablerPractice
+  description: A self-defense weapon that exhausts organic targets, weakening them until they collapse. This one has been undertuned for cadets.
+  components:
+    - type: Tag
+      tags:
+        - Taser
+    - type: Sprite
+      sprite: Objects/Weapons/Guns/Battery/practicedisabler.rsi
```

Comment:
snakecase, also shouldn't this just inherit from WeaponDisabler and change the prototype?

Reply:
the existing naming convention in that folder used both snakecase and not, so I just guessed. I will rename it. The new prototype would only inherit 2 total components if reparented but I will do it real quick.

Reply:
it is done